### PR TITLE
bump bintry versions to latest - 3.1.5 and 2.4.5. Replace .aar style …

### PR DESCRIPTION
**Pull request by 3000**
…integration with gradle in browser sdk

Just a quick update to the android integration docs which havent been bumped in a while.

### CODE REVIEW

**Comment by alexzhaobin** [comment]
👏

**Comment by running-codebase** [comment]
:+1

**Comment by hades-6** [comment]
👏

File: docs/sdk/browser/android/index.md [hunk 1]
```diff
@@ -20,47 +20,19 @@ Sign up to the [Developer Center](https://developer.fillr.com/users/sign_up)
...
    ```
-   ├── project-root
-       ├── app
-       │   └── libs
-       │       └── FillrSDK.aar
-       │   └── src
-       │       └── main
-       ├── build.gradle
-       └── ...
+   compile 'com.fillr:fillr-browser-sdk:2.4.5'
```
Comment (hades-6) [inline docs/sdk/browser/android/index.md]: `compile(group: 'com.fillr', name: 'fillr-browser-sdk', version: '2.4.5', ext: 'aar', classifier: 'fillrBrowserSdkPlusRelease')`

maybe we should add a classifier here? or should we refactor and remove the  variants for the browsersdk?

Comment (3000) [inline docs/sdk/browser/android/index.md]: Have added the classifier!  

Was hoping you would steer me in the right direction for that, thanks! 👍

**Pull request merged**